src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -166,18 +172,24 @@ public function product_id_exists( $product_id ) {
 	 * @return array
 	 */
 	public function get_item_response( $cart_item ) {
-		$product = $cart_item['data'];
+		$product  = $cart_item['data'];
+		$price_dp = wc_get_price_decimals();
+
+		$line_subtotal           = $product->get_price() * wc_stock_amount( $cart_item['quantity'] );
+		$line_total_incl_coupons = isset( $cart_item['line_total'] ) ? $cart_item['line_total'] : $line_subtotal;
```

Comment:
Need to check if we need to convert strings to floats here, and ensure that conversion is handled correctly if using non-standard decimal separators.